addons/website/views/snippets/s_image_hexagonal.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_hexagonal" name="Image Hexagonal">
+    <section class="s_image_hexagonal">
+        <div class="container-fluid">
+            <div class="row o_grid_mode" data-row-count="14">
+                <div class="o_grid_item o_grid_item_image g-col-lg-4 g-height-9 col-lg-4" style="grid-area: 3 / 2 / 12 / 6; z-index: 1;">
+                    <img src="/web_editor/image_shape/website.s_image_hexagonal_default_image_1/web_editor/geometric/geo_hexagon.svg" class="img img-fluid" data-shape="web_editor/geometric/geo_hexagon" data-file-name="s_image_text.jpg" data-original-mimetype="image/jpeg" alt=""/>
+                </div>
+                <div class="o_grid_item g-col-lg-5 g-height-10 col-lg-5" style="grid-area: 3 / 5 / 13 / 9; z-index: 2;">
+                    <h2 class="display-4">Empowering Innovative Solutions</h2>
```

Comment:
I cannot reach `<h2 class="display-4">` from the editor. Should we make this an available option ?

Reply:
I changed it to a regular `<h2 class="display-4-fs">`. We already discussed that here https://github.com/odoo/odoo/pull/177546#discussion_r1734362788

An solution for this would be great in the future, ofc !